packages/patternfly-3/react-console/src/DesktopViewer/MoreInformationContent.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const MoreInformationInstallVariant = ({ os, content }) => (
+  <li>
+    <div className="more-information-install">
+      <b>{os}:</b>
+      {content}
+    </div>
+  </li>
+);
+MoreInformationInstallVariant.propTypes = {
+  os: PropTypes.string.isRequired,
+  content: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired
+};
+
+const MoreInformationContent = () => {
+  const windowsInner = (
+    <div>
+      Download the MSI from{' '}
+      <a href="https://virt-manager.org/download/" target="_blank" rel="noopener noreferrer">
```

Comment:
i think this content should probably be provided as children if we determine to keep it as a PF component.

Reply:
Done. The `<MoreInformationInstallVariant>` content can be provided via both children and a property.